client/keys/delete.go

```diff
@@ -29,7 +29,10 @@ private keys stored in a ledger device cannot be deleted with the CLI.
 		Args: cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			buf := bufio.NewReader(cmd.InOrStdin())
-			clientCtx := client.GetClientContextFromCmd(cmd)
```

Comment:
For commands that only called `GetClientContextFromCmd`, why call now call `GetClientQueryContext`? We don't need to parse the standard flags.

Reply:
`--keyring-dir` is one of the standard query flags, which I think is related to this command. wdyt?

Reply:
For this yes, but I meant more in general. I don't think there is any harm in doing this, I just wanted to point it out :)

Reply:
I was checking the places where we should parse the flags into `clientCtx` and enabled only for places where it makes sense according to me.